dist/LICENSE

```diff
@@ -201,58 +216,6 @@ BSD-3-Clause and Apache-2.0 and MIT and BSD-3-Clause and MIT licenses
 
     github.com/klauspost/compress v1.15.6 BSD-3-Clause and Apache-2.0 and MIT and BSD-3-Clause and MIT
 
-========================================================================
```

Comment:
Are these removed recently?

Reply:
They might relate to https://github.com/apache/skywalking-banyandb/pull/129

Reply:
As I mentioned at https://github.com/apache/skywalking-banyandb/pull/130#discussion_r890748705, if skywalking eye could output coherent results, we can add the license generator to the CI to find these changes ASAP

Reply:
@kezhenxu94 @fgksgf @mrproliu Any suggestion?